adzerk/transform.py

```diff
@@ -92,6 +98,14 @@ def to_collection(spocs):
     return collection
 
 
+def get_topics(body):
+    if body is None:
+        return {}
+    else:
+        p = re.compile('^topic_')
+        return [p.sub('nb_model_', t) for t, v in body.items() if p.match(t) and v in ('true', True)]
```

Comment:
Can we make a note here about why we need a substitution? e.g. is `nb_model_` a prefix we agreed on during the spoc personalization project?

Reply:
They correspond to these model keys in Firefox: https://searchfox.org/mozilla-central/source/browser/app/profile/firefox.js#1313

Your comment made me think about the naming of this field. I decided to rename the field from `topics` to `personalization_models` because I think that's the right level of abstraction. Currently there are only topic models, but in the future there could be others. This way Firefox just needs to match `discoverystream.personalization.modelKeys` to `personalization_models`.

Reply:
Makes a lot of sense. LGTM!